cirq/testing/json.py

```diff
@@ -118,16 +119,17 @@ def all_test_data_keys(self) -> List[str]:
 
 
 def spec_for(module_name: str) -> ModuleJsonTestSpec:
-    import importlib.util
-
-    if importlib.util.find_spec(module_name) is None:
-        raise ModuleNotFoundError(f"{module_name} not found")
+    if not _import.module_exists(module=module_name):
+        raise ModuleNotFoundError(f"{module_name} not found (or it is a namespace package)")
 
     test_module_name = f"{module_name}.json_test_data"
-    if importlib.util.find_spec(test_module_name) is None:
+    if not _import.module_exists(module=test_module_name):
         raise ValueError(
             f"{module_name} module is missing json_test_data package, please set it up."
         )
+
+    import importlib
```

Comment:
What makes this import necessary, and why is it broader than the previous `import importlib.util` when it's used by a smaller section of code?